assets/css/desk-about.css

```diff
@@ -0,0 +1,202 @@
+@import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  font-family: "Lato", sans-serif;
+}
+
+.summit-page {
+  background-color: #f4efef;
+  width: 100%;
+}
+
+.summit-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4rem 0;
+}
+
+.presents {
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-bottom: 5rem;
+}
+
```

Comment:
- [ ] Please ensure that your `texts` on the mobile about home page is aligned at the center as it is on the given design, :slightly_smiling_face: 

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student about texts center](https://github.com/Kwamay/Capstone-Project/assets/43528594/6aefa869-43dd-4ff1-abf4-6aceee22a273)

</td>
      <td>

![design mobile about texts center](https://github.com/Kwamay/Capstone-Project/assets/43528594/888944f5-27aa-4af3-8448-139664f21cc8)

</td>
    </tr>
  </table>